change/@ni-nimble-components-bcea55bf-ed59-4b11-b80b-6311749da460.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Reminder to update the PR description to capture some of the more major changes in the PR.

1. what was copied from FAST and why
2. what changes were made to existing select behavior and why